ports/kf5bookmarks/vcpkg.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "kf5bookmarks",
+  "version-semver": "5.84.0",
```

```suggestion
  "version": "5.84.0",
```
This should be version here.

---

Done.